scripts/ci/changelog/templates/change.md.tera

```diff
@@ -33,10 +33,5 @@
 {%- set repo = "   " -%}
 {%- endif -%}
 
-{%- if c.meta.T and c.meta.T.value == 6 -%}
-{%- set xcm = " [✉️ XCM]" -%}
-{%- else -%}
-{%- set xcm = "" -%}
-{%- endif -%}
```

Comment:
we still have the XCM tag, which is T6-XCM
So if you want to continue using it in the release notes, it's provided

Reply:
ah good catch